domain/host/utils.go

```diff
@@ -97,7 +100,11 @@ func currentHost(ip string, rpcPort int, poolID string) (host *Host, err error)
 
 	routes, err := utils.RouteCmd()
 	if err != nil {
-		glog.V(2).Infof("RouteCmd failed: %s", err)
+		plog.WithError(err).WithFields(log.Fields{
+			"ip": ip,
+			"rpcPort": rpcPort,
+			"poolID": poolID,
+		}).Debug("RouteCmd failed")
```

Comment:
Ditto, although this is debug level so we have more leeway. But try to think of support trying to figure something out based on your message.